Header header
sensor_msgs/PointCloud2 cloud
geometry_msgs/Pose pose
uint16 keyInd
uint16 patchInd
uint8 allPtsKnown
uint8[] dkKnowns
uint8[] inds
